ivy_tests/test_ivy/test_frontends/test_paddle/test_fft.py

```diff
@@ -258,6 +260,40 @@ def test_paddle_ifftshift(
     )
 
 
+@handle_frontend_test(
+    fn_tree="paddle.fft.ihfft2",
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_num_dims=2,
+        min_dim_size=2,
+    ),
+)
+def test_paddle_ihfft2(
+    dtype_x_axis,
+    norm,
+    frontend,
+    backend_fw,
+    test_flags,
+    fn_tree,
+):
+    input_dtypes, x, axis = dtype_x_axis
+    shape = ivy.array(x).shape
+    x = ivy.reshape(x, shape)  # reshape into a 2 dimensional array.
```

Comment:
This should be handled when generating examples in the `dtype_x_axis` argument above

Reply:
Hey @hirwa-nshuti, I have left out the `for norm in ["backward", "forward", "ortho"]` line, and also lines 280, and 281, where I have previously called `ivy` to reshape the input. All 5 tests have passed when I run them locally as well. 

Thank you for your time!